packages/cloud-cognitive/src/components/Datagrid/Datagrid.stories.js

```diff
@@ -312,6 +312,214 @@ export const SelectableRow = () => {
+  const [data] = useState(makeData(20));
+
+  const filters = [
+    {
+      type: 'date',
+      column: 'joined',
+      props: {
+        DatePicker: {
+          datePickerType: 'range',
+        },
+        DatePickerInput: {
+          start: {
+            id: 'date-picker-input-id-start',
+            placeholder: 'mm/dd/yyyy',
+            labelText: 'Joined start date',
+          },
+          end: {
+            id: 'date-picker-input-id-end',
+            placeholder: 'mm/dd/yyyy',
+            labelText: 'Joined end date',
+          },
+        },
+      },
+    },
+    {
+      type: 'number',
+      column: 'visits',
+      props: {
+        NumberInput: {
+          min: 0,
+          id: 'visits-number-input',
+          invalidText: 'A valid value is required',
+          label: 'Visits',
+          placeholder: 'Type a number amount of visits',
+        },
+      },
+    },
+    {
+      type: 'checkbox',
+      column: 'passwordStrength',
+      props: {
+        FormGroup: {
+          legendText: 'Password strength',
+        },
+        Checkbox: [
+          {
+            id: 'normal',
+            labelText: 'Normal',
+            value: 'normal',
+          },
+          {
+            id: 'minor-warning',
+            labelText: 'Minor warning',
+            value: 'minor-warning',
+          },
+          {
+            id: 'critical',
+            labelText: 'Critical',
+            value: 'critical',
+          },
+        ],
+      },
+    },
+    {
+      type: 'radio',
+      column: 'role',
+      props: {
+        FormGroup: {
+          legendText: 'Role',
+        },
+        RadioButtonGroup: {
+          orientation: 'vertical',
+          legend: 'Role legend',
+          name: 'role-radio-button-group',
+        },
+        RadioButton: [
+          {
+            id: 'developer',
+            labelText: 'Developer',
+            value: 'developer',
+          },
+          {
+            id: 'designer',
+            labelText: 'Designer',
+            value: 'designer',
+          },
+          {
+            id: 'researcher',
+            labelText: 'Researcher',
+            value: 'researcher',
+          },
+        ],
+      },
+    },
+    {
+      type: 'dropdown',
+      column: 'status',
+      props: {
+        Dropdown: {
+          id: 'marital-status-dropdown',
+          ariaLabel: 'Marital status dropdown',
+          items: ['relationship', 'complicated', 'single'],
+          label: 'Marital status',
+          titleText: 'Marital status',
+        },
+      },
+    },
+  ];
+
+  const datagridState = useDatagrid(
+    {
+      columns,
+      data,
+      filterProps: {
+        variation: 'flyout',
+        updateMethod: 'batch',
+        primaryActionLabel: 'Apply',
+        secondaryActionLabel: 'Cancel',
+        flyoutIconDescription: 'Open filters',
+        shouldClickOutsideToClose: false,
+        onFlyoutOpen: () => console.log('onFlyoutOpen'),
```

Comment:
This might be a good place to use `action` from storybook instead of `console.log`.
ie:
```
import { action } from '@storybook/addon-actions';
...
...
...
onFlyoutOpen: action('onFlyoutOpen'),
onFlyoutClose: action('onFlyoutClose'),
```

Reply:
Can you also add `emptyStateTitle` and `emptyStateDescription`? I noticed that if I use a filter that doesn't have any results, the only thing that shows up is the empty state illustration.

Reply:
So i initially did that, and nothing happened when I clicked. Let me try to add it back and see if it works 🤷🏻

Reply:
Ahh i always make the mistake of passing it in as () => actions(). Cool that fixed it!